### ISSUE TITLE
Unable to add annotated rule to a composite rule

### ISSUE THREAD

**Issue by afalmeida** [EasyRules/easyrules#41]
hello,
Could you help me please?
I dont understand about compositeRule, when using the method addRule, problems with cast argument.

Ex:
CompositeRule compositeRule =new CompositeRule("compositeRule","",1);
compositeRule.add(new ItemRule());

My ItemRule, I'm using Annotation @Rule.

Thanks

**Comment by benas** [comment]
Hi,

Thank you for pointing out this issue. I've updated the API of CompositeRule to accept annotated objects. It should be working fine new. Here is an [example](https://github.com/EasyRules/easyrules/blob/master/easyrules-core/src/test/java/org/easyrules/core/CompositeRuleTest.java#L118).

I've deployed version 2.3.0-SNAPSHOT with the fix to maven central. Could you please give it a try and tell me if it is ok for you? Many thanks upfront.

Kind regards
Mahmoud

**Comment by afalmeida** [comment]
Hi,
Perfect, version 2.3.0-SNAPSHOT in maven central is not found.
 Thank you very much, 
Kind regards

Anderson

**Comment by benas** [comment]
Hi,
Great.
Have you added this repository:

```xml
<repository>
    <id>ossrh</id>
    <url>https://oss.sonatype.org/content/repositories/snapshots</url>
</repository>
```

to your pom? Instructions to use snapshot versions can be found [here](https://github.com/EasyRules/easyrules#current-version).

Cheers
Mahmoud

**Comment by afalmeida** [comment]
Hi,
It worked, thank you very much, 
Kind regards

Anderson

**Comment by benas** [comment]
Cool. I'll release this fix in the upcoming 2.3 version.
Keep tuned :smile:
